verify_rpms/rpm_verifier.py

```diff
@@ -233,6 +233,30 @@ def aggregate_results(processed_image_list: list[ProcessedImage]) -> dict[str, A
     return aggregated_results
 
 
+def generate_images_processed_result(
+    processed_images_list: list[ProcessedImage], image_url: str, image_digest: str
```

Comment:
I'd drop the `_list`. It's committing to an unnecessary implementation details, while making the arg name unnecessarily longer.
```suggestion
    processed_images: list[ProcessedImage], image_url: str, image_digest: str
```

Reply:
Done